celery/app/task.py

```diff
@@ -784,8 +786,12 @@ def apply(self, args=None, kwargs=None,
                 'routing_key': options.get('routing_key'),
                 'priority': options.get('priority'),
             },
-            'groups': maybe_list(options.get('groups'))
+            'groups': maybe_list(options.get('groups')),
```

Comment:
Why do we still need this if we're expanding all stamped headers?

Reply:
removed, everything is expanding now